src/__test__/components/plots-and-tables/markerHeatmap.test.jsx

```diff
@@ -182,14 +182,14 @@ let loadedProcessingConfigSpy;
 const renderHeatmapPage = async (newStore) => {
   store = createStore(rootReducer, _.cloneDeep(newStore), applyMiddleware(thunk));
 
-  rtl.render(
+  render(
     <Provider store={store}>
       <MarkerHeatmap
         experimentId={experimentId}
       />
     </Provider>,
   );
-  await rtl.waitFor(() => expect(loadConfigSpy).toHaveBeenCalledTimes(1));
+  await waitFor(() => expect(loadConfigSpy).toHaveBeenCalledTimes(1));
 };
 
 describe('Marker heatmap plot', () => {
```

Comment:
the `loads initially` just checks that the load & config have been called right? Isn't it possible to actually test that the user will be seeing a heatmp?

Reply:
test for plot added